src/containers/StreamedTrackCard.tsx

```diff
@@ -0,0 +1,152 @@
+                    <label className="label cursor-pointer">
+                      <span className="label-text mr-2">Low</span>
+                      <input
+                        key={Math.random()}
+                        type="checkbox"
+                        checked={isEncodingActive[0]}
+                        className="checkbox"
+                        onChange={() => {
+                          changeEncoding(trackId, "l", !trackInfo.videoPerks.encodings?.includes("l"));
+                          setEncodingActive([!isEncodingActive[0], isEncodingActive[1], isEncodingActive[2]]);
+                        }}
+                      />
+                    </label>
+                    <label className="label cursor-pointer">
+                      <span className="label-text mr-2">Medium</span>
+                      <input
+                        key={Math.random()}
+                        type="checkbox"
+                        checked={isEncodingActive[1]}
+                        className="checkbox"
+                        onChange={() => {
+                          changeEncoding(trackId, "m", !trackInfo.videoPerks.encodings?.includes("m"));
+                          setEncodingActive([isEncodingActive[0], !isEncodingActive[1], isEncodingActive[2]]);
+                        }}
+                      />
+                    </label>
+                    <label className="label cursor-pointer">
+                      <span className="label-text mr-2">High</span>
+                      <input
+                        key={Math.random()}
+                        type="checkbox"
+                        checked={isEncodingActive[2]}
+                        className="checkbox"
+                        onChange={() => {
+                          changeEncoding(trackId, "l", !trackInfo.videoPerks.encodings?.includes("l"));
+                          setEncodingActive([isEncodingActive[0], isEncodingActive[1], !isEncodingActive[2]]);
+                        }}
+                      />
+                    </label>
+                  </div>
+                )}
+              </div>
+              <div className="flex flex-col">
+                <div className="flex flex-row">
+                  {trackMetadata !== "" && (
+                    <button
+                      className="btn btn-sm m-2 max-w-xs"
+                      onClick={() => {
+                        setTracksId(
+                          tracksId.map((id) => {
+                            if (id?.id === trackId) {
+                              return {
+                                id: trackId,
+                                isMetadataOpened: !id.isMetadataOpened,
+                                audioPerks: id.audioPerks,
+                                videoPerks: id.videoPerks,
+                              };
+                            }
+                            return id;
+                          }),
```

Comment:
Could be an arrow function

```suggestion
                          tracksId.map((id) =>
                            id?.id !== trackId
                              ? id
                              : {
                                  id: trackId,
                                  isMetadataOpened: !id.isMetadataOpened,
                                  audioPerks: id.audioPerks,
                                  videoPerks: id.videoPerks,
                                },
                          )
```

or use guard clause to remove unnecessary nesting

```ts
tracksId.map((id) => {
                            if (id?.id !== trackId) return id;
                            
                            return {
                              id: trackId,
                              isMetadataOpened: !id.isMetadataOpened,
                              audioPerks: id.audioPerks,
                              videoPerks: id.videoPerks,
                            };
                          }),
```